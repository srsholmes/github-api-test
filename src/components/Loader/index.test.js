import React from 'react';
import {render} from '@testing-library/react';
import {Loader} from './index';

describe('Loader', () => {
  it('displays only the loader', () => {
    const {container} = render(<Loader />);
    expect(container).toMatchSnapshot();
  });
});
