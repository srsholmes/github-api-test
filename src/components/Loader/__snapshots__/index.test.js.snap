// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader displays only the loader 1`] = `
<div>
  <div
    class="loader"
  />
</div>
`;
